addons/web/static/tests/core/owl_dialog_tests.js

```diff
@@ -0,0 +1,216 @@
+odoo.define('web.owl_dialog_tests', function (require) {
+"use strict";
+
+const OwlDialog = require('web.OwlDialog');
+
+const makeTestEnvironment = require("web.test_env");
+const testUtils = require("web.test_utils");
+
+const targetClassList = [];
+
+let intercepts = {};
+let target = null;
+
+function getTarget(debug) {
```

Comment:
should be part of testUtils or something...
Same for other helpers

Reply:
ok